Client/Frontend/Browser/Tab Management/Tab.swift

```diff
@@ -786,21 +781,6 @@ class Tab: NSObject {
         }
         return .none
     }
-
-    func updateFaviconCache() {
```

Comment:
Do we have a specific cache for the favicon? Or does it just use kingfisher's default cache

Reply:
Under the hood it uses kingfishers cache for the image and our own cache for the url. It's abstracted in a way that we can swap it out for our own implementation if we find the need to later on. See SiteImageCache for implementation details.